shipyard-templates/shipyard_templates/shipyard_logger.py

```diff
@@ -1,18 +1,46 @@
 import logging
 
 
+def addLoggingLevel(levelName, levelNum, methodName=None):
+    """
```

Comment:
Small change, but these functions and nested functions should be snake case, not camel case